### PR TITLE
Awesome books: a complete website with navigation

### PR DESCRIPTION
**Pull request by ivaan1992**
For this project I: 

- Added style.
- Made dynamic the site.
- Added more code in HTML file.

### CODE REVIEW

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] Good job implementing the three required sections 👍 however according to the requirements you should Display the current date below the navigation bar. For this requirement, you should use a library - [Luxon](https://moment.github.io/luxon/#/install). See the article [Get the current date and time](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time). 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script-class.js [hunk 1]
```diff
@@ -16,3 +16,29 @@ document.querySelector('#addBookButton').addEventListener('click', () => {
   title.value = '';
   author.value = '';
 });
+
+function displayPage(link) {
```
Comment (mikethreels) [inline script-class.js:22]: - [x] Good job implementing the three required sections 👍 however according to the requirements you should Display the current date below the navigation bar. For this requirement, you should use a library - [Luxon](https://moment.github.io/luxon/#/install). See the article [Get the current date and time](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time).

File: index.html [hunk 2]
```diff
@@ -8,20 +8,32 @@
...
+                <span id="monthly">Month</span>
+                <span id="daynumb">00</span>,
+                <span id="year">Year</span>
+            </div>
+            <div class="times">
+                <span id="hours">00</span><span> :</span>
+                <span id="minutes">00</span><span> :</span>
+                <span id="seconds">00</span>
+                <span id="ampm">AM</span>
+            </div> -->
```
Comment (D4nielJ) [inline index.html]: - [x] Let's remove unused pieces of code as this should not be committed to GitHub, instead, we can use GitHub history anytime if we need to review a previous version of our code. [This is considered a best practice, you can read more about it here.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

File: index.html [hunk 3]
```diff
@@ -32,16 +44,26 @@ <h1>Awesome Books</h1>
...
+            <h2>Contact Information</h2>
+            <p>Do you have any questions or you just want to say "Hello"?<br>You can reach out to us!</p>
+            <ul>
+                <li>Our email: mail@mail.com</li>
+                <li>Our phone number: 0045625656565456</li>
+                <li>Our address: Streetname 22, 84503 City, Country</li>
+            </ul>
+        </section>
     </div>
+  </main>
```
Comment (D4nielJ) [inline index.html:55]: [OPTIONAL] I think we could add a footer to the webpage indicating the copyright and information about it as shown in the wireframes to the bottom of our page.

For reference:
image

File: script-class.js [hunk 4]
```diff
@@ -42,3 +43,18 @@ document.querySelector('.menu').querySelectorAll('li').forEach((link) => {
...
+  const DivDate =  document.querySelector('.date-time');
+  DivDate.innerHTML = dt.toLocaleString();  
+}
+
+function initClock() {
+  updateClock();
+  window.setInterval(updateClock, 1);
+}
+
+window.onload = initClock;
```
Comment (D4nielJ) [inline script-class.js]: - [x] Good job implementing your own function to add a clock to the page, unfortunately, part of the requirements of this project is that you use an external library to add function to your webpage, in this case, [Luxon.](https://moment.github.io/luxon/#/install) [You can also read more about working with Luxon here.](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time)

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @ivaan1992, 

Great work on making the changes requested by the previous reviewer(s) 👏

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by er-contreras — APPROVED** [review]
Hi @ivaan1992 and @gdumani , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- I see that you did the work, well done!

Felix

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**